apps/aechannel/test/aesc_fsm_SUITE.erl

```diff
@@ -0,0 +1,193 @@
+    [{nodes, [aecore_suite_utils:node_tuple(N)
+              || N <- [dev1, dev2]]} | Config1].
+
+end_per_suite(_Config) ->
+    ok.
+
+init_per_group(_Group, Config) ->
+    aecore_suite_utils:start_node(dev1, Config),
+    aecore_suite_utils:start_node(dev2, Config),
+    aecore_suite_utils:connect(aecore_suite_utils:node_name(dev1)),
+    ct:log("dev1 connected", []),
+    aecore_suite_utils:connect(aecore_suite_utils:node_name(dev2)),
+    ct:log("dev2 connected", []),
+    try [{initiator, prep_account(initiator, dev1)},
+         {participant, prep_account(participant, dev2)}
+         | Config]
+    catch
+        error:Reason ->
+            Trace = erlang:get_stacktrace(),
+            catch stop_node(dev1, Config),
+            catch stop_node(dev2, Config),
+            error(Reason, Trace)
+    end.
+
+end_per_group(_Group, Config) ->
+    Config1 = stop_node(dev1, Config),
+    _Config2 = stop_node(dev2, Config1),
+    ok.
+
+stop_node(N, Config) ->
+    aecore_suite_utils:stop_node(N, Config),
+    Config.
+
+
+%%%===================================================================
+%%% Test state
+%%%===================================================================
+
+create_channel(Cfg) ->
+    I = ?config(initiator, Cfg),
+    P = ?config(participant, Cfg),
+
+    Port = proplists:get_value(port, Cfg, 9325),
+
+    %% dynamic key negotiation
+    Proto = <<"Noise_NN_25519_ChaChaPoly_BLAKE2b">>,
+
+    Spec = #{initiator          => maps:get(pub, I),
+             participant        => maps:get(pub, P),
+             initiator_amount   => 5,
+             participant_amount => 5,
+             push_amount        => 2,
+             lock_period        => 10,
+             channel_reserve    => 3,
+             minimum_depth      => 3,
+             client             => self(),
+             noise              => [{noise, Proto}],
+             report_info        => true},
+
+    {ok, FsmP} = rpc(dev1, aesc_fsm, participate, [Port, Spec]),
+    {ok, FsmI} = rpc(dev1, aesc_fsm, initiate, ["localhost", Port, Spec]),
+
+    ct:log("FSMs, I = ~p, P = ~p", [FsmI, FsmP]),
+
+    I1 = I#{fsm => FsmI},
+    P1 = P#{fsm => FsmP},
+
+    try await_create_tx_i(I1, P1)
+    catch
+        error:Err ->
+            ct:log("Caught Err = ~p~nMessages = ~p",
+                   [Err, element(2, process_info(self(), messages))]),
+            error(Err, erlang:get_stacktrace())
+    end.
+
+await_create_tx_i(I, P) ->
+    {I1, _} = await_signing_request(create_tx, I),
+    await_funding_created_p(I1, P).
+
+await_funding_created_p(I, P) ->
+    {P1, _} = await_signing_request(funding_created, P),
+    await_funding_signed_i(I, P1).
+
+await_funding_signed_i(I, P) ->
+    receive_info(I, funding_signed),
+    await_funding_locked(I, P).
+
+await_funding_locked(I, P) ->
+    ct:log("mining blocks on dev1 for minimum depth", []),
+    aecore_suite_utils:mine_blocks(aecore_suite_utils:node_name(dev1), 4),
```

Comment:
I see this 4 = 1 + 3, where 
1 = include the multisigned tx on the chain
3 = minimum_depth

Reply:
We need to mine at least one block (hopefully just one, as there's only one tx) in order to get the tx on the chain, then `minimum_depth` blocks after that.

Reply:
so, yes :)